toolkit/tools/imagecustomizerapi/os.go

```diff
@@ -12,22 +12,23 @@ import (
 
 // OS defines how each system present on the image is supposed to be configured.
 type OS struct {
-	ResetBootLoaderType ResetBootLoaderType `yaml:"resetBootLoaderType"`
-	Hostname            string              `yaml:"hostname"`
-	Packages            Packages            `yaml:"packages"`
-	SELinux             SELinux             `yaml:"selinux"`
-	KernelCommandLine   KernelCommandLine   `yaml:"kernelCommandLine"`
-	AdditionalFiles     AdditionalFileList  `yaml:"additionalFiles"`
-	AdditionalDirs      DirConfigList       `yaml:"additionalDirs"`
-	Users               []User              `yaml:"users"`
-	Services            Services            `yaml:"services"`
-	Modules             []Module            `yaml:"modules"`
-	Overlays            *[]Overlay          `yaml:"overlays"`
+	Hostname          string             `yaml:"hostname"`
+	Packages          Packages           `yaml:"packages"`
+	SELinux           SELinux            `yaml:"selinux"`
+	KernelCommandLine KernelCommandLine  `yaml:"kernelCommandLine"`
+	AdditionalFiles   AdditionalFileList `yaml:"additionalFiles"`
+	AdditionalDirs    DirConfigList      `yaml:"additionalDirs"`
+	Users             []User             `yaml:"users"`
+	Services          Services           `yaml:"services"`
+	Modules           []Module           `yaml:"modules"`
+	Overlays          *[]Overlay         `yaml:"overlays"`
+	BootLoader        BootLoader         `yaml:"bootloader"`
```

Comment:
is this change going in before GA or after? This seems to be updated the schema which would break backward compat if after GA...